offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/offers/OfferDbConnector.groovy

```diff
@@ -148,13 +148,17 @@ class OfferDbConnector implements CreateOfferDataSource, FetchOfferDataSource, P
      * @return the id of the stored offer in the database
      */
     private int storeOffer(Offer offer, int projectManagerId, int customerId, int affiliationId){
-        String sqlValues = "VALUE(?,?,?,?,?,?,?,?,?,?,?,?,?)"
+        String sqlValues = "VALUE(?,?,?,?,?,?,?,?,?,?,?,?,?,?)"
         String queryTemplate = OFFER_INSERT_QUERY + " " + sqlValues
         def identifier = offer.identifier
         List<Integer> generatedKeys = []
         Connection connection = connectionProvider.connect()
         log.info("New offer with id: ${offer.identifier}")
         connection.withCloseable {
+
+            String experimentalDesign = null
+            if(offer.experimentalDesign.isPresent()) experimentalDesign = offer.experimentalDesign.get()
+
```

Comment:
Do we want to store `null` in the database? I think it is fine but just wanted to make sure whether we leave it completely optional in the database or not.

If this is intended. Great job 👍

Groovy also provides the nifty feature of `orElse`. Don't know if you want to consider using it therefore here a nit-picky code suggestion.
```suggestion
            String experimentalDesign = offer.experimentalDesign.orElse(null)
```